routes/account.js

```diff
@@ -611,6 +784,14 @@ module.exports = function (
 
   if (isProduction) {
     delete routes[0].config.validate.payload.preVerified
+
+    for (var i = 0; i < routes.length; ++i) {
+      var route = routes[i];
+      if (route.path === '/account/lock') {
+        routes = routes.splice(i, 1)
```

Comment:
@rfk - you are right on the `splice` call, good catch.

A declarative approach could be taken as well - add a `developmentOnly: true` field to the route, only add those routes if in fact in development mode?